connectors/sources/tests/fixtures/oracle/fixture.py

```diff
@@ -5,16 +5,23 @@
 #
 """Oracle module responsible to generate records on the Oracle server.
 """
+import asyncio
 import os
 import random
 import string
 
-import cx_Oracle
+import oracledb
+
+from connectors.utils import RetryStrategy, retryable
 
 DATA_SIZE = os.environ.get("DATA_SIZE", "small").lower()
 _SIZES = {"small": 5, "medium": 10, "large": 30}
 NUM_TABLES = _SIZES[DATA_SIZE]
 
+# If the Oracle Service takes too much time to respond, modify the following retry constants accordingly.
```

Comment:
should these be config options?

Reply:
As mentioned E2E tests are performed by developers So we kept these attributes constant and users can update whenever they want as it depends on the system configuration.